src/basis-cash/BasisCash.ts

```diff
@@ -303,6 +322,19 @@ export class BasisCash {
     }
   }
 
+  async MIC2earned(poolName: ContractName, account = this.myAccount): Promise<BigNumber> {
+    const pool = this.contracts[poolName];
+    try {
+      return await pool.rewards(account);
```

Comment:
The rewards are distributed every week for 10 weeks.  There's also an adjustment on the locking rewards.
I believe getRedeemableReward() * 0.25  would be the actual amount that users receive.  Would you mind testing that again?

Reply:
> The rewards are distributed every week for 10 weeks. There's also an adjustment on the locking rewards.
> I believe getRedeemableReward() * 0.25 would be the actual amount that users receive. Would you mind testing that again?

Thanks for this. I believe I have fixed the problem by using getRedeemableReward(), for some reason, I missed this function and thought it wasnt present in the locked rewards contract. I tested and it appears accurate.

Reply:
I think the actual amount that users received would be 25% of the value returning from `getRedeemableReward()`
Here's an example of this: https://etherscan.io/tx/0xbc5b455d9d8d2463df5e365c804861b2306201406c8a53872d632b6328083667
I believe the reward was adjusted through `feeDistributor`.
That's a bit hack on the rewards though. Not quite sure whether you want to keep it this way to be more transparent (as this is the value returning from the contract) or make it more intuitively for users.

Reply:
Understood. I have made the change.